src/components/shared/editable-detail/editable-detail.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from '../../../test-support/test-utils';
+import EditableDetail from './editable-detail';
+import { EditModeContext } from '../../projects/wbs-details/work-package-container/work-package-container';
+
+describe('Rendering Editable Detail Component', () => {
+  it('renders the content with edit mode disabled', () => {
+    render(
+      <EditModeContext.Provider value={false}>
+        <EditableDetail
+          title="testTitle"
+          value="testValue"
+          readOnly={false}
+          suffix="testSuffix"
+          type="text"
+        />
+      </EditModeContext.Provider>
+    );
```

Comment:
if this is being repeated a bunch with only 1 or 2 values changing, then let's refactor it out into a separate `renderComponent()` function like here: https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/dca5d7b78d63c608c9cac301fdd9b27361f35cd4/src/components/change-requests/change-requests.test.tsx#L49